### PR TITLE
Updating hello wolfi workshop kit

### PR DESCRIPTION
**Pull request by erikaheidi**
This PR updates the Hello Wolfi workshop kit ([#1894](https://github.com/chainguard-dev/edu/issues/1894)).

**UPDATE Dec 10** - after many failed attempts at updating the demo, I talked to @ltagliaferri and we decided it would be best to sunset this resource's instructions and keep only the video for historical reasons. It would not be viable to update the full workshop kit, and we have another, more up-to-date resource on wolfi package building, that is now linked in the disclaimer note that I added to the article.

There are 2 other things in this PR:
- a `note` shortcode to facilitate adding notes using the same styling
- updated the timestamps in the "dev vs prod images" article since it was breaking the hugo build locally for me

### Shortcode Usage

```
{{< note >}}
Note content
{{< /note >}}
```

Preview: https://deploy-preview-1926--ornate-narwhal-088216.netlify.app/open-source/wolfi/hello-wolfi/

### CODE REVIEW

File: content/open-source/wolfi/hello-wolfi.md [hunk 1]
```diff
@@ -121,19 +121,17 @@ With both the PHP and Composer dependencies in place, you can now build the appl
 On Linux systems, use the following command:
 
 ```shell
-docker run --privileged --rm -v "${PWD}":/work -- \
-  cgr.dev/chainguard/melange build melange-app.yaml \
-  --arch x86_64 \
-  --signing-key melange.rsa --keyring-append melange.rsa.pub
+docker run --privileged --rm -v ${PWD}:/work cgr.dev/chainguard/melange \
```
Comment (SharpRake) [inline content/open-source/wolfi/hello-wolfi.md:124]: This command fails for me, I start getting this error:

```
. . .

2024/11/22 20:03:35 INFO running step "Build Minicli application"
2024/11/22 20:03:35 INFO PHP's phar extension is missing. Composer requires it to run. Enable the extension or recompile php without --disable-phar then try again. name="Build Minicli application"
2024/11/22 20:03:35 INFO deleting guest dir /tmp/melange-guest-1475690852
2024/11/22 20:03:35 INFO deleting workspace dir /tmp/melange-workspace-3361137084
2024/11/22 20:03:35 INFO removing image path /tmp/melange-guest-2907343915
2024/11/22 20:03:35 ERRO failed to build package: unable to run package hello-minicli pipeline: unable to run pipeline: exit status 1
```

There's always a chance this is just something I've messed up on my side, let me know if I can provide any more info on what's going on here

Comment (SharpRake) [reply content/open-source/wolfi/hello-wolfi.md:124]: FYI the remaining commands all fail after this as well

**Review by SharpRake — COMMENTED** [review]
Things started going haywire for me in Step 5. I'm getting an error and I'm not sure how to interpret it, let me know if I can provide any more info.

File: content/open-source/wolfi/hello-wolfi.md (hunk 1 above)
Comment (erikaheidi) [reply content/open-source/wolfi/hello-wolfi.md:124]: Ok, let me check everything again, it seems some extension is missing

**Review by SharpRake — APPROVED** [review]
:+1:

**Pull request merged**